Sofascore/Sofascore/EventViewController.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+import SofaAcademic
+import SnapKit
+import UIKit
+
+class EventsViewController: UIViewController {
+    
+    private var sectionsData: [Section] = []
+    private let eventRespository = EventRepository(dataSource: MockData())
+    private var collectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.minimumInteritemSpacing = 0
+        layout.minimumLineSpacing = 0
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        return collectionView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addViews()
+        styleViews()
+        setupConstraints()
+        fetchEvents()
+    }
+    
+    func fetchEvents() {
+        eventRespository.fetch { sections in
+            if let sections {
+                self.sectionsData = sections
+            }
+        }
+    }
+}
+
+extension EventsViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return sectionsData.count
```

Comment:
```suggestion
       sectionsData.count
```

U ovakvim situacijama, kada je oneliner je return redudantan. Isprobaj